server/controllers/userAuth.js

```diff
@@ -448,6 +448,27 @@ const deleteAccount = async (req, resp) => {
   return resp.send();
 };
 
+const updateEmailNotificationSettings = async (req, resp) => {
+  const { never, weekly } = req.body;
+  const { _id } = req.user;
+
+  try {
+    await User.findOneAndUpdate(
+      { _id },
+      {
+        emailNotificationSettings: {
```

Comment:
I wonder if this should be called more general settings and inside we should save more specific "data"